eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/README.md

```diff
@@ -4,227 +4,124 @@ languages:
+These messages are published to one `Event Hub` instance and then consumed by one consumer 
+endpoint from the same application.
 
 ## Getting started
 
 Running this sample will be charged by Azure. You can check the usage and bill at 
-[this link][azure-account].
+[here][azure-account].
 
+### Create Azure resources
 
+1. Create [Azure Event Hubs][create-event-hubs].
+    Please note `Basic` tier is unsupported. After creating the Azure Event Hub, you
+    can create your own Consumer Group or use the default "$Default" Consumer Group.
 
-### Create Azure resources
+2. Create [Azure Storage][create-azure-storage] for checkpoint usage.
+
+### Configuration credential options
 
 We have several ways to config the Spring Cloud Stream Binder for Azure
-Event Hub. You can choose anyone of them.
+Event Hubs. You can choose anyone of them.
 
 >[!Important]
 >
 >  When using the Restful API to send messages, the **Active profiles** must contain `manual`.
 
-
 #### Method 1: Connection string based usage
 
-1.  Create [Azure Event Hubs][create-event-hubs].
-    Please note `Basic` tier is unsupported. After creating the Azure Event Hub, you 
-    can create your own Consumer Group or use the default "$Default" Consumer Group.
-
-1.  Create [Azure Storage][create-azure-storage] for checkpoint use.
-
 1.  Update [application.yaml][application.yaml].
-
     ```yaml
     spring:
       cloud:
         azure:
-          eventhub:
-            # Fill event hub namespace connection string copied from portal
-            connection-string: [eventhub-namespace-connection-string] 
-            # Fill checkpoint storage account name, access key and container 
-            checkpoint-storage-account: [checkpoint-storage-account]
-            checkpoint-access-key: [checkpoint-access-key]
-            checkpoint-container: [checkpoint-container]
+          eventhubs:
+            connection-string: ${AZURE_EVENTHUBS_CONNECTION_STRING}
+            processor:
+              checkpoint-store:
+                container-name: ${AZURE_STORAGE_CONTAINER_NAME}
+                account-name: ${AZURE_STORAGE_ACCOUNT_NAME}
+                account-key: ${AZURE_STORAGE_ACCOUNT_KEY}
         stream:
           function:
             definition: consume;supply
           bindings:
             consume-in-0:
-              destination: [eventhub-name]
-              group: [consumer-group]
+              destination: ${AZURE_EVENTHUB_NAME}
+              group: ${AZURE_EVENTHUB_CONSUMER_GROUP}
+              consumer:
+                batch-mode: false # false is default value
             supply-out-0:
-              destination: [the-same-eventhub-name-as-above]
+              destination: ${SAME_AS_ABOVE_DESTINATION}
+          eventhubs:
+            bindings:
+              consume-in-0:
+                consumer:
+                  checkpoint:
+                    mode: MANUAL # RECORD | MANUAL
```

Comment:
Not only RECORD AND MANUAL mode is supported, you can choose either give all modes or delete this comment.

Reply:
updated